main.go

```diff
@@ -31,6 +31,14 @@ func main() {
 		log.Fatalf("couldn't open configuration file: %s\n", err)
 	}
 
+	if errors := localCfg.Validate(); len(errors) > 0 {
+		for _, err := range errors {
+			log.Println(err)
+		}
+
+		os.Exit(1)
+	}
```

Comment:
Could you also add a new test scenario for this new feature?

A similar test can be seen [here](https://github.com/preslavmihaylov/todocheck/blob/master/testing/todocheck_test.go#L322)

To run the tests locally:
```
go test -v -count=1
```

Reply:
Thanks! I'll take care of it.

Reply:
Done.